composer.json

```
@@ -24,20 +24,20 @@
     "contao-community-alliance/dc-general": "~2.1",
     "contao-community-alliance/meta-palettes": "^1.11 || ^2.0",
     "contao/core-bundle": "^4.4",
-    "doctrine/dbal": "^2.6",
+    "doctrine/dbal": "^2.6 || ^3.3",
     "menatwork/contao-multicolumnwizard-bundle": "^3.4.0",
     "metamodels/attribute_select": "~2.1",
     "metamodels/core": "~2.1",
     "netzmacht/contao-leaflet-maps": "~3.1",
     "netzmacht/contao-toolkit": "~3.0",
     "netzmacht/php-javascript-builder": "^1.0.2",
     "netzmacht/php-leaflet": "^1.0",
-    "symfony/config": "^3.4 || ^4.0",
-    "symfony/dependency-injection": "^3.4.26 || ^4.1.12",
-    "symfony/event-dispatcher": "^3.4 || ^4.0",
-    "symfony/http-kernel": "^3.4 || ^4.0",
-    "symfony/routing": "^3.4 || ^4.0",
-    "symfony/translation": "^3.4 || ^4.0"
+    "symfony/config": "^5.4",
+    "symfony/dependency-injection": "^5.4",
+    "symfony/event-dispatcher": "^5.4",
+    "symfony/http-kernel": "^5.4",
+    "symfony/routing": "^5.4",
+    "symfony/translation": "^5.4"
```

```suggestion
    "symfony/config": "^4.4 || ^5.4",
    "symfony/dependency-injection": "^4.4 || ^5.4",
    "symfony/event-dispatcher": "^4.4 || ^5.4",
    "symfony/http-kernel": "^4.4 || ^5.4",
    "symfony/routing": "^4.4 || ^5.4",
    "symfony/translation": "^4.4 || ^5.4"
```
Please still allow symfony `^4.4`, as long as Contao 4.9 is in security support